src/Admin/MaintainAuthUsers/components/AuthUser.js

```diff
@@ -121,6 +122,11 @@ const AuthUser = props => {
               <Table.Cell>{contextUser.verified ? 'Yes' : 'No'}</Table.Cell>
               <Table.Cell>&nbsp;</Table.Cell>
             </Table.Row>
+            <Table.Row>
+              <Table.CellHeader>Last logged in</Table.CellHeader>
+              <Table.Cell>{moment(contextUser.lastLoggedIn).format('DD MMMM YYYY - HH:mm:ss')}</Table.Cell>
```

Comment:
might be fine - Andy made us remove leading 0 on day in use-of-force, not if this is a blanket thing